boards/shields/mikroe_accel13_click/mikroe_accel13_click.overlay

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2023 Mark Olsson <mark@markolsson.se>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
please add an alias for iis2dlpc_mikroe_accel13_click --> accel0 (see other shields)

Reply:
I've done this now...I didn't add anything to the docs though, I wasn't sure what to write and where.